securedrop-export/debian/changelog-buster

```diff
@@ -1,3 +1,9 @@
+securedrop-export (0.3.0+bullseye) unstable; urgency=medium
```

Comment:
These should say buster for now, the script automagically does `s/buster/bullseye/` but not in the other direction.

Reply:
Thanks!  Fixed in 950602bf6f574217539075b98c280ca637968d81.